<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>900</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>-1</y>
      <width>641</width>
      <height>841</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Core">
     <attribute name="title">
      <string>Core</string>
     </attribute>
     <widget class="QLabel" name="lbl_process">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Top Processes</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="disp_process">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>380</y>
        <width>631</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri','Consolas'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;Please Refresh.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="but_refresh_process">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>350</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="disp_basic_info">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>591</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Please restart the program.</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_basic_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Basic System Information</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Devices">
     <attribute name="title">
      <string>Devices</string>
     </attribute>
     <widget class="QTextBrowser" name="disp_inp_dev">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>570</y>
        <width>631</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_inp_dev">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>530</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input Devices</string>
      </property>
     </widget>
     <widget class="QPushButton" name="but_refresh_inp_dev">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>540</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_RAM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RAM Usage</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_CPU">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_storage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Primary Storage</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="bar_cpu_usage">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="bar_hdd_used">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>440</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="bar_ram_load">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>260</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="disp_cpu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>441</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="disp_ram">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="disp_hdd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>431</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="but_refresh_dev_info">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="lbl_cpu_load">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:14pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:400;&quot;&gt;CPU Load&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="lbl_ram_used">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>290</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:14pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:400;&quot;&gt;RAM Used&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="lbl_storage_used">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>470</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:14pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:400;&quot;&gt;Storage Used&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
